### PR TITLE
Added Loader

### LINKED ISSUE #45: Feat: Website Preloader

### PR DESCRIPTION
**Pull request by GenDelta**
I added the following to the code: 
- Added a loader 
image

Closes #45 
Kindly review the changes,

### CODE REVIEW

File: src/components/Loader/index.css [hunk 1]
```diff
@@ -0,0 +1,796 @@
+/*=================
+Basic style
+==================*/
+html, body{
+    position: relative;
+    width: 100%;
+    height: 100%;
+    overflow: hidden;
+    background-color: #212022;
```
Comment (codexsit) [inline src/components/Loader/index.css]: use class/id and not tag selector

**Review by codexsit — APPROVED** [review]
LGTM!

**Review by codexsit — CHANGES_REQUESTED** [review]
made suggested changes

**Comment by Miran-Firdausi** [comment]
Different Pre-loader implemented in #65

**Pull request closed without merging**